libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1721,7 +1725,9 @@ bool AP_Arming::disarm(const AP_Arming::Method method, bool do_disarm_checks)
         }
     }
 #endif
-
+#if defined(HAL_ARM_GPIO_PIN)
+    hal.gpio->write(HAL_ARM_GPIO_PIN,HAL_ARM_GPIO_POL_INVERT ? 1 : 0);
```

Comment:
use update_arming_pin(), so boardconfig is checked on both arm and disarm

Reply:
done